src/Microsoft.DotNet.Interactive.Kql/KqlKernelConnector.cs

```diff
@@ -35,7 +35,15 @@ public async Task<Kernel> CreateKernelAsync(string kernelName)
 
         var connectionDetails = await BuildConnectionDetailsAsync();
 
-        var client = new ToolsServiceClient(PathToService);
+        var serviceArgs = $"--parent-pid {Environment.ProcessId}";
```

Comment:
This parent-pid argument here is the ID of the current process, so that we can close the service process after the parent process has exited. This was added to the SQL kernel a while ago, but was missed in the Kusto kernel.